eole/config/models.py

```diff
@@ -670,11 +691,11 @@ def encoder_decoder_type(cls, data: Any) -> Any:
         if not (isinstance(data, dict)):
             return data
         if "encoder" in data.keys():
-            data["encoder"]["encoder_type"] = "transformer"
+            data["encoder"].encoder_type = "transformer"
         else:
             data["encoder"] = {"encoder_type": "transformer"}
         if "decoder" in data.keys():
-            data["decoder"]["decoder_type"] = "transformer"
+            data["decoder"].decoder_type = "transformer"
```

Comment:
Why was this change needed?

Reply:
not sure why it popped but for sure data["encoder"] is not a dict anymore I think this should have been changed some time ago.